setup.py

```diff
@@ -711,117 +713,64 @@ def run(self):
 main_libraries = ['shm']
 main_link_args = CAFFE2_LIBS + [NANOPB_STATIC_LIB, PROTOBUF_STATIC_LIB]
 main_sources = [
-    "torch/csrc/PtrWrapper.cpp",
-    "torch/csrc/Module.cpp",
-    "torch/csrc/Generator.cpp",
-    "torch/csrc/Size.cpp",
-    "torch/csrc/Dtype.cpp",
+    "torch/csrc/DataLoader.cpp",
```

Comment:
this list has been `M-x sort-lines`ed, and then all files which are in the TORCH_SRCS list below in the cmake file have been removed